### ISSUE TITLE
arm64 make failed

### ISSUE THREAD

**Issue by syzhang42** [anrieff/libcpuid#205]
My operating system and architecture information is as follows：
![image](https://github.com/user-attachments/assets/1b2372cc-8e71-470c-b27c-c58c362591bc)
I use make to compile the source code, and I get the following error.
![image](https://github.com/user-attachments/assets/b1c74eec-bc24-4925-a55a-747a2fcc9baf)
What should I do? Help me please.

**Comment by zacikpa** [comment]
What glibc version are you running? `HWCAP_CPUID` was added [into kernel](https://github.com/torvalds/linux/commit/77c97b4ee21290f5f083173d957843b615abbff2) in version 4.11 and [into glibc](https://sourceware.org/git/?p=glibc.git;a=blobdiff;f=sysdeps/unix/sysv/linux/aarch64/bits/hwcap.h;h=cef3225af00fbd0b8025bbcdbb6b202a9d1d8b28;hp=59758945210e1c1e31bad38818db5aa5a987b785;hb=d2e4346a30683cc42c57bd1bfd457897d78c6d7e;hpb=fbe355fbd1973d6e29561084b3eaeb4bfe9d515a) in version 2.26.

@TheTumultuousUnicornOfDarkness, there should probably be an additional check [here](https://github.com/anrieff/libcpuid/blob/bcd2dea9ee9246f9b89c0619b0047d8eafa36627/libcpuid/cpuid_main.c#L1299C28-L1299C29):
```c
# if defined(HAVE_GETAUXVAL) && defined(HWCAP_CPUID) /* Linux */
	return (getauxval(AT_HWCAP) & HWCAP_CPUID);
```

**Comment by TheTumultuousUnicornOfDarkness** [comment]
Thank you for your answer @zacikpa. Fixed in 7770fffb6a380d991e152344b17dbf51c6961de0.
